react/src/Participants/components/ParticipantInfoDialog.tsx

```diff
@@ -96,20 +125,31 @@ export default function ParticipantInfoDialog(props: DialogProp) {
                         </div>
                     </>
                 )}
-                {analyses.length > 0 && (
-                    <>
-                        <Divider />
-                        <div className={classes.infoSection}>
-                            <InfoList
-                                infoList={getAnalysisInfoList(analyses)}
-                                title="Analyses"
-                                enums={enums}
-                                icon={<ShowChart />}
-                                linkPath="/analysis"
-                            />
-                        </div>
-                    </>
-                )}
+
+                <>
+                    <Divider />
+                    <Box margin={3}>
+                        <Button
+                            variant="contained"
+                            size="small"
+                            onClick={() => setAnalysisInfoListOpen(!analysisInfoListOpen)}
+                        >
+                            {`${analysisInfoListOpen ? "Hide" : "Show"} Analyses`}
```

Comment:
We should display how many analyses there are in the button label (eg. "Show Analyses (3)")